app/views/applications/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<header id="page-header">
+  <h2> New Adoption Application </h2>
+</header>
+
+<main>
+  <section>
+    <h3> Select favorited pets you wish to adopt </h3>
+    <%= form_tag '/applications' do %>
+      <% @favorites.each do |favorite| %>
+        <%= check_box_tag 'favorite_ids[]', favorite.id, false, :id => "checkbox-#{favorite.id}" %>
+        <%= label_tag favorite.name %>
```

Comment:
I think this label_tag is still not formally connected to the check_box_tag because they don't have the same `name:` attribute, but I'm not sure how to fix that because our checkboxes have a shared name.